tasks/clean.unit.test.js

```diff
@@ -28,8 +28,8 @@ describe('Clean task', () => {
         paths: ['custom/location/here/**/*']
       }
     ]
-  )('cleans destination "$destination"', async ({ destination, paths }) => {
-    jest.mock('./task-arguments.js', () => ({ destination }))
+  )('cleans destination "$destination"', async ({ destination: mockDestination, paths }) => {
+    jest.mock('./task-arguments.js', () => ({ destination: mockDestination }))
```

Comment:
We need to follow the `babel-jest` approach of prefixing imports with "mock" when we _don't_ want them automatically hoisting to the top of the _transpiled_ file

This doesn't need documenting as the CLI will explain this to you

The Jest guidance on [Using with ES module imports](https://jestjs.io/docs/manual-mocks#using-with-es-module-imports) was useful